cgmes/cgmes-import-postprocessor/src/test/java/com/powsybl/cgmes/importer/postprocessor/TransformerConversionTest.java

```diff
@@ -0,0 +1,266 @@
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author Luma Zamarreño {@literal <zamarrenolm at aia.es>}
+ * @author José Antonio Marqués {@literal <marquesja at aia.es>}
+ */
+class TransformerConversionTest {
+
+    private static final String EQUALS_LINE = "======================";
+
+    @Test
+    void miniBusBranchPhaseAngleClock() {
+        Conversion.Config config = new Conversion.Config();
+        List<CgmesImportPostProcessor> postProcessors = new ArrayList<>();
+        postProcessors.add(new PhaseAngleClock());
+
+        Network n = networkModel(CgmesConformity1Catalog.miniBusBranch(), config, postProcessors);
+
+        boolean ok = t2xCompareFlow(n, "f1e72854-ec35-46e9-b614-27db354e8dbb", -318.691633, 1424.484145, 436.204160, 1393.367311);
+        assertTrue(ok);
+        ok = t3xCompareFlow(n, "5d38b7ed-73fd-405a-9cdb-78425e003773", -7.505045, -1.896561, -288.380946, 1216.566903, 351.090362, 1199.878285);
+        assertTrue(ok);
+    }
+
+    @Test
+    void miniBusBranchPhaseAngleClockZero() {
+        Conversion.Config config = new Conversion.Config();
+        List<CgmesImportPostProcessor> postProcessors = new ArrayList<>();
+        postProcessors.add(new PhaseAngleClock());
+
+        Network n = networkModel(CgmesConformity1ModifiedCatalog.miniBusBranchPhaseAngleClockZero(), config, postProcessors);
+
+        boolean ok = t2xCompareFlow(n, "f1e72854-ec35-46e9-b614-27db354e8dbb", -0.087780, -0.178561, 0.087782, 0.178613);
+        assertTrue(ok);
+        ok = t3xCompareFlow(n, "5d38b7ed-73fd-405a-9cdb-78425e003773", -0.000001, -0.000022, 0.000002, 0.000068, -0.000001, -0.000045);
+        assertTrue(ok);
+    }
+
+    @Test
+    void miniBusBranchT2xPhaseAngleClock1NonZero() {
+        Conversion.Config config = new Conversion.Config();
+        List<CgmesImportPostProcessor> postProcessors = new ArrayList<>();
+        postProcessors.add(new PhaseAngleClock());
+
+        Network n = networkModel(CgmesConformity1ModifiedCatalog.miniBusBranchT2xPhaseAngleClock1NonZero(), config, postProcessors);
+
+        boolean ok = t2xCompareFlow(n, "f1e72854-ec35-46e9-b614-27db354e8dbb", -318.691633, 1424.484145, 436.204160, 1393.367311);
+        assertTrue(ok);
+    }
+
+    @Test
+    void miniBusBranchT3xAllPhaseAngleClockNonZero() {
+        Conversion.Config config = new Conversion.Config();
+        List<CgmesImportPostProcessor> postProcessors = new ArrayList<>();
+        postProcessors.add(new PhaseAngleClock());
+
+        Network n = networkModel(CgmesConformity1ModifiedCatalog.miniBusBranchT3xAllPhaseAngleClockNonZero(), config, postProcessors);
+
+        boolean ok = t3xCompareFlow(n, "5d38b7ed-73fd-405a-9cdb-78425e003773", -1494.636083, 1530.638656, 981.686099, 1826.870720, 562.199867, 309.289551);
+        assertTrue(ok);
+    }
+
+    private boolean t2xCompareFlow(Network n, String id, double p1, double q1, double p2, double q2) {
```

Comment:
Can you try to share the following methods with their original versions to limit the code duplication?